HW6/nataliia/t3.py

```diff
@@ -0,0 +1,11 @@
+def char_frequency(str):
```

Comment:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result